media/client/main/source/ClientController.cpp

```diff
@@ -44,15 +44,22 @@ ClientController::ClientController(const std::shared_ptr<IControlIpcFactory> &Co
 {
     RIALTO_CLIENT_LOG_DEBUG("entry:");
 
-    const char commitID[] = COMMIT_ID;
+    const char srcRev[] = SRCREV;
 
-    if (std::strlen(commitID) > 0)
+    if (std::strlen(srcRev) > 0)
     {
-        RIALTO_CLIENT_LOG_MIL("Commit ID: %s", commitID);
+        if (srcRev[0] == 'v')
+        {
+            RIALTO_CLIENT_LOG_MIL("Release Tag(s): %s", srcRev);
```

Comment:
Right now I think that we could actually do something like RIALTO_CLIENT_LOG_MIL("Commit ID %s(%s)", srcRev, tag);